docs/crio.8.md

```diff
@@ -212,6 +213,8 @@ crio [GLOBAL OPTIONS] command [COMMAND OPTIONS] [ARGUMENTS...]
 
 **--enable-tracing**: Enable OpenTelemetry trace data exporting
 
+**--evented-pleg**: Enable CRI-O to generate the container pod-level events in order to optimize the performance of the Pod Lifecycle Event Generator (PLEG) module in Kubelet.
```

Comment:
I'm not sure If I'm in favor of manually enabling/disabling the feature. What is the reason for not automatically decide that?

Reply:
According to the following [discussion](https://kubernetes.slack.com/archives/C03HA69L6R1/p1657179643142429?thread_ts=1657126350.580129&cid=C03HA69L6R1), to be on sync with the feature gate in kubelet, a config switch in containerd, this has been added in CRI-O.

cc: @harche @rphillips